internal/server/v0/statvarsummary/golden/get_statvar_summary/females.json

```diff
@@ -1091,24 +1091,8 @@
                   "minValue": 0,
                   "maxValue": 72821
                 },
-                "Place": {
```

Comment:
The deletion is suspicious

Reply:
This is likely because the recent NCES school district addition perhaps fixed up the type in infrequent or something, so that Place type doesn't show up.

To confirm, one of the listed places still has the same Count_Person_Female stats: https://autopush.datacommons.org/browser/geoId/sch2000005?statVar=Count_Person_Female